### ISSUE TITLE
Question about the setting in Table 5

### ISSUE THREAD

**Issue by dneirfi** [tensorflow/tpu#981]
Hi.
Thanks for sharing your nice work!

ViLD evaluates the LVIS-trained model on COCO validation set in table 5 in paper.
The LVIS-trained model means trained with only base classes in LVIS? or the all classes in LVIS?

Thanks
